### DISCUSSION TITLE
How can we verify the subscription

### DISCUSSION THREAD

**Discussion by sjkarthik** [opencrvs/opencrvs-core#6400] [Technical Q&A]
We are getting subscription api response as 202 but the birth registration event is not received to opencrvs side mediator for MOSIP. Could you please help us on this.

**Comment by rikukissa** [comment]
Hi @sjkarthik ! 

Just to clarify your situation, could you answer the following questions:
- Can you share the payload used when you register the webhook? 
- If I understand correctly, no request from OpenCRVS is received to this URL. Is this correct?
- Are you using this [mosip-mediator](https://github.com/opencrvs/mosip-mediator) ? More about the state of that repository you can [read here](https://github.com/opencrvs/opencrvs-core/discussions/6395#discussioncomment-7990769).
- Which version of OpenCRVS are you using?

**Comment by euanmillar** [comment]
This discussion is occurring here: https://github.com/opencrvs/opencrvs-core/discussions/6395
